internal/mocks/mock_default_mongodb_version.go

```diff
@@ -0,0 +1,48 @@
+// Code generated by MockGen. DO NOT EDIT.
+// Source: github.com/mongodb/mongocli/internal/store (interfaces: DefaultVersionGetter)
+
+// Package mocks is a generated GoMock package.
+package mocks
```

Comment:
I am not sure why I have this mock file to add

Reply:
I may have forgotten to generate mocks when added that method